src/actions.js

```diff
@@ -71,8 +70,8 @@ const WORKFLOWS_FULL_PROJECTION = () => [
   'group',
 ];
 
-export function fetchBenefitPlans(modulesManager, params) {
```

Comment:
It's better not do do this kind of changes in long living code, if `fetchBenefitPlans` would be used in other modules we're unaware of then functionality could break.